components/preparing_workspace/preparing_workspace.scss

```diff
@@ -53,12 +53,14 @@
 
     .tertiary-button {
         @include tertiary-button;
+        @include button-medium;
```

Comment:
Are these changes intended here?

Reply:
Yes, it is needed to specify styling according to size as well. As mentioned in other comments, for the next iteration of this (Signup) plan to review the option to have a button wrapper component, so that type and sizes could be set via props.

Reply:
Yes, the styles for mixin `tertiary-button` were updated, so need the size ones added here.

Reply:
Gotcha, thanks for clarifying!